cmd/mindthegap/push/imagebundle/image_bundle.go

```diff
@@ -74,40 +79,47 @@ func NewCommand(out output.Output) *cobra.Command {
 			}()
 			out.EndOperation(true)
 
-			// do not include the scheme
-			destRegistry := destRegistryURI.Address()
-
-			skopeoRunner, skopeoCleanup := skopeo.NewRunner()
-			cleaner.AddCleanupFn(func() { _ = skopeoCleanup() })
+			logs.Debug.SetOutput(out.V(4).InfoWriter())
+			logs.Warn.SetOutput(out.InfoWriter())
+
+			var remoteOpts []remote.Option
+			insecure := flags.SkipTLSVerify(destRegistrySkipTLSVerify, destRegistryURI)
+			if insecure || destRegistryCACertificateFile != "" {
```

Comment:
This all looks correct but I launched a local insecure registry with `docker run -d -p 5000:5000 --restart=always --name registry registry:2`

And seeing:
```
➜  mindthegap git:(jimmi/skopeo-to-crane) ✗ dist/mindthegap_darwin_amd64_v1/mindthegap push image-bundle --to-registry http://kubernetes.docker.internal:5000 --image-bundle ~/Desktop/crane.tar --to-registry-insecure-skip-tls-verify
 ✓ Creating temporary directory
 ✓ Unarchiving image bundle "/Users/dkoshkin/Desktop/crane.tar"
 ✓ Parsing image bundle config
 ✓ Starting temporary Docker registry
 ✗ Copying docker.io/apteno/alpine-jq:2021-01-19 (from bundle) to kubernetes.docker.internal:5000/apteno/alpine-jq:2021-01-19
Get "https://kubernetes.docker.internal:5000/v2/": http: server gave HTTP response to HTTPS client
```

Pushing the same bundle to the same registry with a binary built from `main` works
```
➜  mindthegap git:(main) ✗ dist/mindthegap_darwin_amd64_v1/mindthegap push image-bundle --to-registry http://kubernetes.docker.internal:5000 --image-bundle ~/Desktop/crane.tar --to-registry-insecure-skip-tls-verify
 ✓ Creating temporary directory
 ✓ Unarchiving image bundle "/Users/dkoshkin/Desktop/crane.tar"
 ✓ Parsing image bundle config
 ✓ Starting temporary Docker registry
 ✓ Copying docker.io/apteno/alpine-jq:2021-01-19 (from bundle) to kubernetes.docker.internal:5000/apteno/alpine-jq:2021-01-19
 ...
 ```

Reply:
Hmm this looks like some weirdness with (my) docker setup.

`http://0.0.0.0:5000` fails with both versions, `http://127.0.0.1:5000` and `http://localhost:5000` both work with this new version.

Reply:
Im not even sure what `kubernetes.docker.internal` is TBH.

Reply:
```
➜  mindthegap git:(jimmi/skopeo-to-crane) ✗ dist/mindthegap_darwin_amd64_v1/mindthegap push image-bundle --to-registry http://localhost:5000 --image-bundle ~/Desktop/skopeo.tar --to-registry-insecure-skip-tls-verify
 ✓ Creating temporary directory
 ✓ Unarchiving image bundle "/Users/dkoshkin/Desktop/skopeo.tar"
 ✓ Parsing image bundle config
 ✓ Starting temporary Docker registry
 ✓ Copying docker.io/apteno/alpine-jq:2021-01-19 (from bundle) to localhost:5000/apteno/alpine-jq:2021-01-19
 ✓ Copying docker.io/bitnami/kubectl:1.17.16 (from bundle) to localhost:5000/bitnami/kubectl:1.17.16
 ✓ Copying docker.io/bitnami/kubectl:1.19.7 (from bundle) to localhost:5000/bitnami/kubectl:1.19.7
 ✓ Copying docker.io/bitnami/kubectl:1.21.3 (from bundle) to localhost:5000/bitnami/kubectl:1.21.3
 ✓ Copying docker.io/bitnami/memcached:1.6.9-debian-10-r114 (from bundle) to localhost:5000/bitnami/memcached:1.6.9-debian-10-r114
 ✓ Copying docker.io/bitnami/postgresql:11.11.0-debian-10-r62 (from bundle) to localhost:5000/bitnami/postgresql:11.11.0-debian-10-r62
 ✓ Copying gcr.io/google_containers/kubernetes-dashboard-init-amd64:v1.0.0 (from bundle) to localhost:5000/google_containers/kubernetes-dashboard-init-amd64:v1.0.0
 ✓ Copying ghcr.io/banzaicloud/fluentd:v1.12.4-alpine-4 (from bundle) to localhost:5000/banzaicloud/fluentd:v1.12.4-alpine-4
 ✓ Copying k8s.gcr.io/kube-state-metrics/kube-state-metrics:v1.9.8 (from bundle) to localhost:5000/kube-state-metrics/kube-state-metrics:v1.9.8
 ✓ Copying k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.1.1 (from bundle) to localhost:5000/kube-state-metrics/kube-state-metrics:v2.1.1
 ✓ Copying quay.io/jetstack/cert-manager-cainjector:v1.5.4 (from bundle) to localhost:5000/jetstack/cert-manager-cainjector:v1.5.4
 ```
 ```
➜  mindthegap git:(jimmi/skopeo-to-crane) ✗ dist/mindthegap_darwin_amd64_v1/mindthegap push image-bundle --to-registry http://127.0.0.1:5000 --image-bundle ~/Desktop/skopeo.tar --to-registry-insecure-skip-tls-verify
 ✓ Creating temporary directory
 ✓ Unarchiving image bundle "/Users/dkoshkin/Desktop/skopeo.tar"
 ✓ Parsing image bundle config
 ✓ Starting temporary Docker registry
 ✓ Copying docker.io/apteno/alpine-jq:2021-01-19 (from bundle) to 127.0.0.1:5000/apteno/alpine-jq:2021-01-19
 ✓ Copying docker.io/bitnami/kubectl:1.17.16 (from bundle) to 127.0.0.1:5000/bitnami/kubectl:1.17.16
 ✓ Copying docker.io/bitnami/kubectl:1.19.7 (from bundle) to 127.0.0.1:5000/bitnami/kubectl:1.19.7
 ✓ Copying docker.io/bitnami/kubectl:1.21.3 (from bundle) to 127.0.0.1:5000/bitnami/kubectl:1.21.3
 ✓ Copying docker.io/bitnami/memcached:1.6.9-debian-10-r114 (from bundle) to 127.0.0.1:5000/bitnami/memcached:1.6.9-debian-10-r114
 ✓ Copying docker.io/bitnami/postgresql:11.11.0-debian-10-r62 (from bundle) to 127.0.0.1:5000/bitnami/postgresql:11.11.0-debian-10-r62
 ✓ Copying gcr.io/google_containers/kubernetes-dashboard-init-amd64:v1.0.0 (from bundle) to 127.0.0.1:5000/google_containers/kubernetes-dashboard-init-amd64:v1.0.0
 ✓ Copying ghcr.io/banzaicloud/fluentd:v1.12.4-alpine-4 (from bundle) to 127.0.0.1:5000/banzaicloud/fluentd:v1.12.4-alpine-4
 ✓ Copying k8s.gcr.io/kube-state-metrics/kube-state-metrics:v1.9.8 (from bundle) to 127.0.0.1:5000/kube-state-metrics/kube-state-metrics:v1.9.8
 ✓ Copying k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.1.1 (from bundle) to 127.0.0.1:5000/kube-state-metrics/kube-state-metrics:v2.1.1
 ✓ Copying quay.io/jetstack/cert-manager-cainjector:v1.5.4 (from bundle) to 127.0.0.1:5000/jetstack/cert-manager-cainjector:v1.5.4
```

Reply:
localhost is special... i didn't test against an http registry not on localhost. do we need to support that?

Reply:
>do we need to support that?

It didn't work before so totally fine that it still doesn't work. I remember you and I looking at `localhost` on my Mac and we did find it was special.